build.gradle

```diff
@@ -59,8 +62,8 @@ wrapper {
 }
 
 compileJava {
-    sourceCompatibility = 1.7
-    targetCompatibility = 1.7
+    sourceCompatibility = 1.8
+    targetCompatibility = 1.8
```

Comment:
I'm okay with replacing the minimum requirement from Java 7 to Java 8. Can you change this in README.md as well (first line)?

Reply:
Updated, see https://github.com/web-push-libs/webpush-java/pull/139/commits/6dd7db820661648797f717b39fe914e0039a0106.